src/SFA.DAS.AssessorService.Application.Api.Client/Clients/StandardVersionClient.cs

```diff
@@ -23,7 +23,23 @@ public async Task<IEnumerable<StandardVersion>> GetAllStandardVersions()
                 return await RequestAndDeserialiseAsync<IEnumerable<StandardVersion>>(request, $"Could not get the list of standards");
             }
         }
-        
+
+        public async Task<IEnumerable<StandardVersion>> GetLatestStandardVersions()
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"/api/v1/standard-version/standards/latest"))
+            {
+                return await RequestAndDeserialiseAsync<IEnumerable<StandardVersion>>(request, $"Could not get the list of latest standards");
+            }
+        }
+
+        public async Task<IEnumerable<StandardVersion>> GetStandardVersionsByIFateReferenceNumber(string iFateReferenceNumber)
```

Comment:
I think this can be removed as well?

Reply:
This method and the related api methods are used by 565 so I will need to restore the api methods previously removed